README.md

```diff
@@ -288,11 +288,18 @@ left out of this section.
   * Performs some actions to prevent the platform from being detected as a bot.
   * Note, these aren't comprehensive and automated interaction with the site
     will still appear very bot-like.
-* `headless`
-  * Launch the browser in headless mode (supported as of Firefox 56),
-    no GUI will be visible.
-  * Use this when running browsers on a remote machine or to run crawls in the
-      background on a local machine.
+* `display_mode`:
+  * `native`:
+    * Launch the browser normally - GUI will be visible
+  * `headless`:
+    * Launch the browser in headless mode (supported as of Firefox 56),
+        no GUI will be visible.
+    * Use this when running browsers on a remote machine or to run crawls in the
+        background on a local machine.
+  * `xvfb`:
+    * Launch the browser using the X virtual frame buffer. In this mode, Firefox
+      is not running in it's own headless mode, but no GUI will be displayed.
+    * This mode only works on Linux.
```

Comment:
We should mention here _why_ someone might want to run in `xvfb`. Maybe:

```
Firefox running in xvfb may more accurately reflect a real computing
environment than Firefox's native headless mode (e.g., for reasons like 
[Bug 1375585](https://bugzilla.mozilla.org/show_bug.cgi?id=1375585)).
Use this mode if `headless` mode doesn't support an API you'd like to 
monitor, or if you're concerned about bot detection.
```

Reply:
I think many aspects of bot detection will be the same / alternate problems in xvfb on a server vs headless mode somewhere else. I will add a note about the problems with headless, and open an issue to document the relative merits of xvfb vs headless and we can discuss there.

Reply:
Thanks. That's fine as well. I mainly want to make it possible for someone without context to decide between the two modes based on the differences we currently know about.